contrib/IECoreUSD/src/IECoreUSD/PointInstancerAlgo.cpp

```diff
@@ -108,16 +122,42 @@ IECore::ObjectPtr readPointInstancer( pxr::UsdGeomPointInstancer &pointInstancer
 
 	// Prototype paths
 
+	const static bool gafferUSDPointInstancersRelativeProtoypes = checkEnvFlag( "GAFFERUSD_POINTINSTANCER_RELATIVEPROTOTYPES", true );
```

Comment:
Cortex shouldn't know about Gaffer so I think this should be `IECOREUSD_POINTINSTANCER_RELATIVE_PROTOTYPES`. We can have the Gaffer wrapper set that automatically based on the value of the Gaffer env-var.